xtask/src/tasks.rs

```diff
@@ -198,7 +198,7 @@ pub fn serial_term() -> color_eyre::Result<()> {
         }
     };
 
-    let mut port = serialport::open(&dongle.port_name)?;
+    let mut port = serialport::new(&dongle.port_name, 9600).open()?;
```

Comment:
9600 is pretty low, would you mind trying this with `115200`, which is a pretty standard speed as well?

Reply:
9600 was the default speed for `serialport::open` (I checked :D )
115200 works for `dongle-flash` and `serial-term` (actually now I get regular packets of `received 45 bytes (CRC=Ok(0xc0ba), LQI=15)`; previously I only saw the double `now listening on channel 20` from troubleshooting)

Reply:
Let's switch it over, we can beta test in the training tomorrow, and hotpatch it back quickly if we need to :)

Reply:
already did!